src/app/pages/movieinfo/[id].js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'next/router'
+
+import { getMovieInfo } from '../../actions/movie-info';
+
+import MovieInfo from '../../containers/movie/MovieInfo';
+import MovieList from '../../containers/movie/MovieList';
+import Footer from '../../components/common/Footer';
+import { moviesListGoodSelector } from '../../selectors';
+
+export class MoviePage extends React.Component {
+
+  constructor(props){
+    super(props);
+  }
+
+  getMovieId() {
+    const href = window.location.href;
+    const last = href.lastIndexOf('/');
+    return href.slice(last + 1);
```

Comment:
loading of initial data should be performed in getInitialProps static method (https://nextjs.org/docs/api-reference/data-fetching/getInitialProps) and there you will receive context with path and query params.
Example how to prefetch redux data https://github.com/kirill-konshin/next-redux-wrapper#pagegetinitialprops